secure_message/repository/database.py

```diff
@@ -57,7 +58,7 @@ def set_from_domain_model(self, domain_model):
         self.survey = domain_model.survey
         self.collection_exercise = domain_model.collection_exercise
 
-    def serialize(self, user, body_summary=False):  # pylint:disable=too-complex
+    def serialize(self, user, body_summary=False):  # pylint:disable=too-complex, too-many-branches
```

Comment:
hmm .. not sure adding to the pylint is correct thing to do here. I'm actually surprised we have too-complex disabled.

This method looks fair to complicated tbh and should be refactored.

Reply:
Fair comment. Will do . It was added when we had several hundred pylint errors and we wanted to get to zero.